ldap_auth_provider.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, config, account_handler):
             )
 
         self.ldap_mode = config.mode
-        self.ldap_uri = config.uri
+        self.ldap_uri = [config.uri] if isinstance(config.uri, str) else config.uri
```

Comment:
Well, that is up to you. I would keep only one.

Reply:
* you can't have an option called `uri` which contains a list of uris
* you can't drop support for the existing `uri` option

=> you will need to support both, as alternatives.

Reply:
>     * you can't have an option called `uri` which contains a list of uris
You can. It wouldn't be without precedent:
- python LDAP3: server: the Server object to be contacted. It can be a ServerPool.
- php: ldap_uri
- postfix: server_host
- openvpn: url
- mediawiki: server
- simplesamlphp: hostname

Reply:
ok fair enough, keep the name of the config setting as-is.

But please rename the `ldap_uri` property.

Reply:
Sure, done and readme updated.